tests/Integration/Behaviour/Features/Context/Domain/Product/Combination/CombinationListingFeatureContext.php

```diff
@@ -296,9 +296,7 @@ private function assertListedCombinationsProperties(array $expectedDataRows, arr
                 'Unexpected attributes count in combination'
             );
 
-            if (empty($expectedCombination['image url'])) {
-                Assert::assertNull($editableCombinationForListing->getImageUrl(), 'Unexpected combination image');
-            } else {
+            if (!empty($expectedCombination['image url'])) {
```

Comment:
Is it normal to have a space in your array key?

Reply:
Yes because it's related to behats tests:
https://github.com/PrestaShop/PrestaShop/pull/26506/files#diff-bba1a8f95cf00b64c0d5da9e1e43fe50920ed8c110fc435012e4e588d71b8649R71

@jolelievre correct me if I'm wrong

Reply:
Yep exactly, we don't have a strict convention for this But it matches the column name
Some prefer snake case, some space-separated words (close to human-readable or PM point of view), but since no styling is imposed for now it's acceptable